docs/using-client-keystone-auth.md

```diff
@@ -108,6 +108,11 @@ arguments
 (`--keystone-url`, `--domain-name`, `--user-name`, `--password` and `--project-name`), respectively.
 If they are not specified, the user will be prompted to enter them at the time of the interactive
 session.
+It is also possible to authenticate with [Keystone Application Credentials](https://docs.openstack.org/keystone/queens/user/application_credentials.html).
```

Comment:
suggest to use https://docs.openstack.org/keystone/latest/user/application_credentials.html
so we always recommend latest

Reply:
You are right ! Fixed